### PR TITLE
D129 - Consistent benthic attribute colors in site pie charts 

### PR DESCRIPTION
**Pull request by tienhoah**

### CODE REVIEW

**Comment by tienhoah** [comment]
#129

File: src/components/CardChartContent.js [hunk 1]
```diff
@@ -5,10 +5,11 @@ import Box from '@material-ui/core/Box';
 import BarChart from './BarChart';
 import PieChart from './PieChart';
 
+import PropTypes from 'prop-types';
```
Comment (brianbancroft) [inline src/components/CardChartContent.js]: Remember that the absolute imports should always precede the relative ones. This should be near the top

File: src/components/CardChartContent.js [hunk 2]
```diff
@@ -28,4 +28,12 @@ const CardChartContent = ({
...
 };
 
+CardChartContent.propTypes = {
+  chartType: PropTypes.string,
+  pieChartContent: PropTypes.object,
+  setToPrivate: PropTypes.bool,
+  privateLabel: PropTypes.string,
+  histogramContent: PropTypes.array
+};
+
```
Comment (brianbancroft) [inline src/components/CardChartContent.js]: 🌮 👍

File: src/components/InformationCard.js [hunk 3]
```diff
@@ -141,8 +139,7 @@ InformationCard.propTypes = {
   title: PropTypes.string,
   type: PropTypes.string,
   bleachingSubItems: PropTypes.object,
-  pieChartContent: PropTypes.array,
-  pieChartLegend: PropTypes.object,
+  pieChartContent: PropTypes.object,
```
Comment (brianbancroft) [inline src/components/InformationCard.js]: This might be an opportunity to check out Proptypes.shape

No sweat if you don't want to do it for this round.

**Pull request merged**